configs/projects/puppet-agent.rb

```diff
@@ -241,7 +249,20 @@
   end
   proj.component "ruby-shadow" unless platform.is_aix? || platform.is_windows?
   proj.component "ruby-augeas" unless platform.is_windows?
-  proj.component "openssl"
+
+  if use_system_openssl?
+    if (platform.is_el? || platform.is_fedora?)
```

Comment:
is this going to work reasonably on all of the fedora/el platforms we support or do we need to limit scope more? Fedora we only have relatively recent, and it doesn't look like any of those have moved to openssl 1.1.0 yet, but will PA work with that? Will it work with 0.9.8 installed on EL5?

Just trying to make sure we're explicit where we need to be.